get-started/index.md

```diff
@@ -18,9 +18,9 @@ Follow the steps below to install the Cryostat Operator via [OperatorHub](https:
 The Cryostat Operator requires [cert-manager](https://cert-manager.io/) to run.
 If not already installed in your cluster, please 
 [install](https://cert-manager.io/docs/installation/) it using your preferred method.
-Once installed, proceed with the operator installation steps below.
+Once installed, proceed with the Operator installation steps below.
 
-**Warning**: Although it is possible to [disable cert-manager integration](https://github.com/cryostatio/cryostat-operator/blob/v2/docs/config.md#disabling-cert-manager-integration), it is NOT recommended to
+**Warning**: Although it is possible to [disable cert-manager integration](https://github.com/cryostatio/cryostat-operator/blob/{{site.data.versions.cryostat.release-branch}}/docs/config.md#disabling-cert-manager-integration), it is NOT recommended to
 do so unless cert-manager is unavailable AND one of the following applies to you:
 - You have another solution for encrypting traffic
 - You trust everything running in the same cluster where the Cryostat Operator is deployed
```

Comment:
https://github.com/cryostatio/cryostatio.github.io/blob/1581e061e8cb07eb59cf5865f181d7216fb4da52/get-started/index.md?plain=1#L35

We might need to update the version of OLM here. When using `operator-sdk olm install`, it installed version [`0.24.0`](https://github.com/operator-framework/operator-lifecycle-manager/releases).

To check OLM version installed, I think we could do:

```
kubectl exec pod/olm-operator-59555bc789-27bn7 -n olm -- olm --version
OLM version: v0.24.0
git commit: dac8182eb62acc1cb489d17ccc34f243f43d4f94
```

Reply:
Clicking "Install" [here](https://operatorhub.io/operator/cryostat-operator) also shows 0.24.0 now.

Reply:
Yup, in the second-to-last commit I've updated this.

Reply:
Awesome ^^